feconf.py

```diff
@@ -605,6 +611,8 @@ def get_empty_ratings():
 COLLECTION_UNPUBLISH_PREFIX = '/collection_editor_handler/unpublish'
 COLLECTION_EDITOR_URL_PREFIX = '/collection_editor/create'
 COLLECTION_URL_PREFIX = '/collection'
+COMMUNITY_DASHBOARD_URL = '/community_dashboard'
+COMMUNITY_OPPORTUNITIES_DATA_URL = '/opportunitiessummaryhandler'
```

Comment:
Should this also be separated by _?

Reply:
This won't a user-visible link and we use such pattern for data url.

Reply:
Hmmm, ok